src/netbase.cpp

```diff
@@ -377,7 +377,7 @@ bool Socks5(const std::string& strDest, uint16_t port, const ProxyCredentials* a
     vSocks5.push_back(SOCKSVersion::SOCKS5); // VER protocol version
     vSocks5.push_back(SOCKS5Command::CONNECT); // CMD CONNECT
```

Comment:
Could be a candidate for another PR - since we are changing `SOCKS5Atyp` to an enum  class, would it make sense to also change the following enums to enum classes `SOCKSVersion, SOCKS5Command, SOCKS5Method` and `SOCKS5Reply`?